src/util/preemptive-rate-limit.ts

```diff
@@ -47,6 +48,14 @@ const getRateResetTime = (rateLimitResponse: Octokit.RateLimitGetResponse, log:
 	const resetEpochDateTime = Math.max(rateLimitResponse?.resources?.core?.reset, rateLimitResponse?.resources?.graphql?.reset);
 	// Get the difference in seconds between now and reset time
 	const timeToResetInSeconds = resetEpochDateTime - (Date.now()/1000);
-	log.info({ timeToResetInSeconds }, "Preemptive rate limit reset time");
-	return timeToResetInSeconds;
+	const finalTimeToRestInSeconds = timeToResetInSeconds <= 0 ? DEFAULT_PREEMPTY_RATELIMIT_DELAY_IN_SECONDS : timeToResetInSeconds;
+
+	log.info({
+		timeToResetInSeconds,
+		finalTimeToRestInSeconds,
+		coreReset: rateLimitResponse?.resources?.core?.reset,
```

Comment:
Why just don't log the whole object?

Reply:
I feel this should be enough. But I don't have a strong reason either way though.